crates/cxx-qt-lib/src/core/mod.rs

```diff
@@ -48,6 +54,9 @@ pub use qstring::QString;
 mod qstringlist;
 pub use qstringlist::QStringList;
 
+mod qt;
+pub use qt::{AspectRatioMode, CaseSensitivity, DateFormat, SplitBehaviorFlags};
```

Comment:
how should we handle these they are normally namespaced to `Qt::AspectRatioMode`?

Reply:
I think this approach is fine.
Instead of namespacing with `Qt::AspectRatioMode`, it would be `cxx_qt_lib::AspectRatioMode` or `cxx_qt_lib::qt::AspectRationMode`.
So basically, keep "top-level" Qt types top-level.